Carbon.Registry/Functions/Revoke-CRegistryPermission.ps1

```diff
@@ -0,0 +1,63 @@
+
+function Revoke-CRegistryPermission
+{
+    <#
+    .SYNOPSIS
+    Revokes *explicit* permissions on a file, directory, registry key, or certificate's private key/key container.

```

Comment:
Is this supposed to mention file, directory, and certificate paths (i.e. non registry paths)?